src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/TestFixtures/DatabaseHelper.cs

```diff
@@ -299,10 +299,10 @@ public IEnumerator<object[]> GetEnumerator()
         {
             foreach (string connStrAE in DataTestUtility.AEConnStrings)
             {
-                yield return new object[] { connStrAE, @"ExecuteReader", 1 };
-                yield return new object[] { connStrAE, @"ExecuteReader", 3 };
-                yield return new object[] { connStrAE, @"ExecuteNonQuery", 1 };
-                yield return new object[] { connStrAE, @"ExecuteNonQuery", 3 };
+                yield return new object[] { connStrAE, @"ExecuteReader" };
```

Comment:
If we remove the `1` and `3` parameters, aren't we basically saying run the test twice for `ExecuteReader` and `ExecuteNonQuery`?

... actually these tests are really a mess. I want to clean them up so bad but I just don't have the bandwidth to do it :(